package.json

```diff
@@ -20,7 +18,7 @@
     "prepush": "node scripts/testOnlyCheck.js",
     "prerelease": "node scripts/release.js 1.2.0 1.3.0",
     "start": "npm run dev",
-    "start:https": "cross-env SSL=true npm run dev",
+    "start:https": "npm run dev -- --server-type https",
```

Comment:
I chose to use the parameter here but we could have used an environment variable and set it in the config https://webpack.js.org/configuration/dev-server/#devserverserver